### PR TITLE
React Calculator App - Milestone 2

### PR DESCRIPTION
**Pull request by 300ms**
- Create Button, ButtonPanel, App, and Display components.
- Add a little bit of styling to make it seem good.

### CODE REVIEW

File: src/components/App.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import Display from './Display';
+import ButtonPanel from './ButtonPanel';
+
+export default function App() {
+  return (
+    <div className="app-container">
```
Comment (whiz25) [inline src/components/App.js]: Wrap the Display and ButtonPanel in React.Fragment shorthand.
`<>
   ...
</>
`

**Review by whiz25 — CHANGES_REQUESTED** [review]
## changes required 🙏🏿 🙏🏿 🙏🏿 
Nice work on this milestone. You have satisfied most of the requirements. Well done.

### changes 🔕 🔕 🔕 
- check comments for suggestions

### Martin Afani
#### Slack: whiz25

File: src/components/App.js (hunk 1 above)
Comment (300ms) [reply src/components/App.js]: Wrapped the App and ButtonPanel components in React.Fragment as requested. Display has only one node to return, so I did not use React Fragments in Display component.

**Review by whiz25 — APPROVED** [review]
## approved 👍🏿👍🏿👍🏿
Nice work making the requested change. Merge and proceed to the next milestone.

### Martin Afani
#### Slack: whiz25

**Pull request merged**